server/src/main/java/org/elasticsearch/index/fielddata/fieldcomparator/BytesRefFieldComparatorSource.java

```diff
@@ -104,6 +105,11 @@ protected SortedDocValues getSortedDocValues(LeafReaderContext context, String f
 
         return new FieldComparator.TermValComparator(numHits, null, sortMissingLast) {
 
+            @Override
+            public LeafFieldComparator getLeafComparator(LeafReaderContext context) throws IOException {
+                return super.getLeafComparator(context);
+            }
```

Comment:
Any reason to add this override that I might be missing?

Reply:
no, thanks for catching it, it was a leftover of some other experiment :)